app/presenters/documents_info_export_presenter.rb

```diff
@@ -0,0 +1,86 @@
+class DocumentsInfoExportPresenter
+  attr_reader :lead_organisation, :type
+
+  def initialize(lead_organisation:, type:)
+    @lead_organisation = lead_organisation
+    @type = type
+  end
+
+  def call
+    document_ids.sort.map do |id|
```

Comment:
I'd suggest this sort would be more intuitive as part of the data query (and hopefully can be part of the pagination)

Reply:
Yup done.